docs/vidu-docs/参考生视频.md

```
@@ -0,0 +1,186 @@
+| aspect_ratio       |           | String        | 可选  | 比例默认 16:9，可选值如下：16:9、9:16、1:1<br>注：q2模型 支持任意宽高比              |
+| resolution         |           | String        | 可选  | 分辨率参数，默认值依据模型和视频时长而定：<br>viduq3-turbo（3-16秒）：默认 720p, 可选：540p、720p、1080p<br>viduq3（3-16秒）：默认 720p, 可选：540p、720p、1080p<br>viduq2、viduq2-pro ：默认 720p, 可选：540p、720p、1080p<br>viduq1 ：默认 1080p, 可选：1080p<br>vidu2.0 ：默认 360p, 可选：360p、720p |
+| movement_amplitude |           | String        | 可选  | 运动幅度默认 auto，可选值：auto、small、medium、large<br>注：使用q2、q3模型时该参数不生效 |
+| payload            |           | String        | 可选  | 透传参数不做任何处理，仅数据传输注：最多 1048576个字符                              |
+| off_peak           |           | Bool          | 可选  | 错峰模式，默认为：false，可选值：<br>- true：错峰生成视频；<br>- false：即时生成视频；<br>注1：错峰模式消耗的积分更低，具体请查看产品定价<br>注2：错峰模式下提交的任务，会在48小时内生成，未能完成的任务会被自动取消，并返还该任务的积分；<br>注3：您也可以[手动取消](https://platform.vidu.cn/docs/cancel-task-api)错峰任务<br>注4：q3模型在audio参数为true时，支持错峰任务；q2、q1、2.0系列模型在audio为false时支持错峰任务 |
+| watermark          |           | Bool          | 可选  | 是否添加水印<br>- true：添加水印；<br>- false：不添加水印；<br>注1：目前水印内容为固定，内容由AI生成，默认不加<br>注2：您可以通过watermarked_url参数查询获取带水印的视频内容，详情见查询任务接口 |
+| wm_position        |           | Int           | 可选  | 水印位置，表示水印出现在图片的位置，默认为：3，可选项为：<br>1：左上角<br>2：右上角<br>3：右下角<br>4：左下角 |
+| wm_url             |           | String        | 可选  | 水印内容，此处为图片URL不传时，使用默认水印：内容由AI生成                              |
+| meta_data          |           | String        | 可选  | 元数据标识，json格式字符串，透传字段，您可以 自定义格式 或使用 示例格式 ，示例如下：<br>{<br>"Label": "your_label","ContentProducer": "yourcontentproducer","ContentPropagator": "your_content_propagator","ProduceID": "yourproductid", "PropagateID": "your_propagate_id","ReservedCode1": "yourreservedcode1", "ReservedCode2": "your_reserved_code2"<br>}<br>该参数为空时，默认使用vidu生成的元数据标识 |
+| callback_url       |           | String        | 可选  | Callback 协议<br>需要您在创建任务时主动设置 callback_url，请求方法为 POST，当视频生成任务有状态变化时，Vidu 将向此地址发送包含任务最新状态的回调请求。回调请求内容结构与查询任务API的返回体一致<br>回调返回的"status"包括以下状态：<br>- processing 任务处理中<br>- success 任务完成（如发送失败，回调三次）<br>- failed 任务失败（如发送失败，回调三次）<br>Vidu采用回调签名算法进行认证，详情见：[回调签名算法](https://platform.vidu.cn/docs/callback-signature) |
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)